README.md

```diff
@@ -1,2 +1,391 @@
 # ada-interface
```

Comment:
I'll push a dedicated commit onto the issue_7 branch with changes to the README and request your review separately.